### PR TITLE
Preserve local config during update

### PR DESCRIPTION
**Pull request by elrido**
During the last update from 2021.07 to 2021.09 my [local.config.php got replaced](https://github.com/friendica/docker/issues/166#issuecomment-934713008). I found that the archive that gets downloaded from https://files.friendi.ca/ contained such file, while it is not present in the friendica git tree.

This change will exclude that file from the configuration file update. It also amends a path used in the installation to point to the unpacked archive.

Is it correct to only change this in the template scripts in the top level or should this also get changed in the scripts for each of the docker images (i.e. `2021.09/apache/entrypoint.sh`)? I'm not sure on the process/script to refresh these.

### CODE REVIEW

**Comment by nupplaphil** [comment]
Please execute `./update.sh` after changing the template and commit every changed file as well :) 

But I think this PR fixes not the root cause.. I think I have to recreate the stable archive again @tobiasd :( what do you say? In the meantime I'll add this change to have a running image again

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -74,7 +74,7 @@ if expr "$1" : "apache" 1>/dev/null || [ "$1" = "php-fpm" ]; then
     rsync $rsync_options --delete --exclude-from=/upgrade.exclude /usr/src/friendica/ /var/www/html/
 
     # Update docker-based config files, but never delete other config files
-    rsync $rsync_options --update /usr/src/friendica/config/ /var/www/html/config/
+    rsync $rsync_options --update --exclude=/local.config.php /usr/src/friendica/config/ /var/www/html/config/
```
Comment (MrPetovan) [inline docker-entrypoint.sh]: This needs to exclude `addon.config.php` as well.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by tobiasd** [comment]
@nupplaphil can you regenerate the release archive plus the sig and the sum files?

File: docker-entrypoint.sh (hunk 1 above)
Comment (elrido) [reply docker-entrypoint.sh]: Thank you, added it.

**Comment by elrido** [comment]
Thank you all for the guidance. Did apply the change to all the images via update.sh as well.

**Review by MrPetovan — APPROVED** [review]
Looks good to me, @nupplaphil ?

**Comment by nupplaphil** [comment]
merged, but I'll recreate the archive without the config as well to fix the root cause..

**Pull request merged**